app/styles/components/freestyle-palette-item.scss

```diff
@@ -9,9 +9,9 @@ Hey look... these are `markdown` notes:
 END-FREESTYLE-USAGE */
 
 // BEGIN-FREESTYLE-USAGE fpi
-$freestylePaletteItemBorderColor: #cecece;
-$FreestylePaletteItem-backgroundColor: #fff;
-$FreestylePaletteItem-color: #808080;
+$freestylePaletteItemBorderColor: #cecece !default;
+$FreestylePaletteItem-backgroundColor: #fff !default;
+$FreestylePaletteItem-color: #2F4F4F !default;
```

Comment:
I'm surprised sass-lint didn't break the build here. Should be all lowercase... `#2f4f4f`